pages/windows/color.md

```diff
@@ -0,0 +1,15 @@
+# color
+
+> Set the console foreground and background colors.
+
+- Set the console colors to the default values:
+
+`color`
+
+- Set the console foreground and background to a specific color:
+
+`color {{color_hex}}`
```

Comment:
Is the "#" needed ? I suggest `color {{hex_code}}`

Reply:
I guess it's slightly misleading, it's not a standard hex colour. It's 2 hex digits.

0 - F are different colours and a combination of 2 of them (such as `70` or `F3`) is what you provide.